includes/polling/os/cambium.inc.php

```diff
@@ -20,6 +20,10 @@
     $masterSlaveMode = ucfirst(snmp_get($device, 'masterSlaveMode.0', '-Oqv', "CAMBIUM-PTP250-MIB"));
     $hardware = 'PTP 250 '.$masterSlaveMode;
     include 'includes/polling/wireless/cambium-250.inc.php';
+} elseif (strstr($cambium_type, 'PTP 58600')) {
+    $masterSlaveMode = ucfirst(snmp_get($device, 'masterSlaveMode.0', '-Oqv', "CANOPY-SYS-MIB"));
+    $hardware = 'PTP 58600 '.$masterSlaveMode;
+    include 'includes/polling/wireless/motorola58600.inc.php';
```

Comment:
Is this supposed to be there?

Reply:
Hopefully not hijacking but Cambium PTP 58500 are showing as generic

Reply:
@YukonRob this is expected. We'll need to add support for the 58500, free free to open a new issue and add the requested files and we can do that for you.